python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -33,3 +55,10 @@ cdef class KafkaDatasource(Datasource):
     cdef string delimiter
 
     cdef datasource* get_datasource(self) nogil
+
+    cpdef bool commit_offset(self,
+                             string topic,
+                             int32_t partition,
+                             int64_t offset)
```

Comment:
Should this be made `void` as well to align with the C++ class?

Reply:
yes, I build locally and it didn't fail which has me confused though? I do agree it needs to be changed though